src/components/Icons.js

```diff
@@ -55,6 +27,27 @@ ArrowRight.defaultProps = {
   width: "56",
 }
 
+export const ArrowNorthEast = ({ size = 19, color = "#0A0806" }) => (
+  <svg
+    width={size}
+    height={size}
+    fill="none"
+    viewBox="0 0 19 19"
+    aria-labelledby="arrow-north-east-title"
+  >
+    <title id="arrow-north-east-title">North East Arrow Icon</title>
+    <path
+      fill={color}
+      d="M13.04 13.02l-.058-5.883-8.083 8.082-1.118-1.118 8.082-8.083L5.98 5.96l-.02-1.6 8.7-.02-.02 8.7-1.6-.02z"
+    ></path>
+  </svg>
+)
+
+ArrowNorthEast.propTypes = {
+  color: PropTypes.string,
+  size: PropTypes.number,
+}
```

Comment:
Just a small note:

<img width="115" alt="Screen Shot 2020-05-27 at 08 32 47" src="https://user-images.githubusercontent.com/8245/83019271-a8bcbf00-9ff4-11ea-95a0-863408861a74.png">

The vertical alignment here is a bit off. I wonder if we don't continue to use the CSS-based character for this, which gets some alignment assistance from the font, rather than a custom SVG arrow.

Reply:
Will fix! I originally used a unicode character for this, but arrow didn't look the same. There was one that looked closer to the design, but it isn't supported the the standard set of characters.

Reply:
May be worth checking in with Liz and figuring out whether the tradeoff is worth it.